addons/hr_recruitment/tests/test_website_job_parsing.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.hr.tests.common import TestHrCommon
+from odoo.modules.module import get_module_resource
+
+
+class TestWebsiteJobParsing(TestHrCommon):
+    """ Test recruitment process through job posting websites such as LinkedIn and ICTJobs """
+
+    def test_00_linkedin_application_parsing(self):
+        self.dep_rd = self.env['hr.department'].create({
```

Comment:
line 12 - 28 could potentially be grouped into a common test setup, so that you don't have to repeat this code between the two test methods. It doesn't matter too much cause it's so little but it might make it more obvious what the method actually tests at first glance.